test/interactions/keyInteractionTests.ts

```diff
@@ -101,43 +72,127 @@ describe("Interactions", () => {
         keyInteraction.onKeyPress(aCode, aCallback);
         $target.simulate("keydown", { keyCode: aCode });
         assert.isTrue(aCallbackCalled, "callback for \"a\" was properly connected back to the interaction");
-
-        svg.remove();
+      });
+
+      it("multiple keyPress callbacks are possible", () => {
+          var aCallback1Called = false;
+          var aCallback1 = () => aCallback1Called = true;
+          var aCallback2Called = false;
+          var aCallback2 = () => aCallback2Called = true;
+
+          keyInteraction.onKeyPress(aCode, aCallback1);
+          keyInteraction.onKeyPress(aCode, aCallback2);
+
+          TestMethods.triggerFakeMouseEvent("mouseover", component.background(), 100, 100);
+          $target.simulate("keydown", { keyCode: aCode });
+          assert.isTrue(aCallback1Called, "callback 1 for \"a\" was called when \"a\" key was pressed");
+          assert.isTrue(aCallback1Called, "callback 2 for \"b\" was called when \"a\" key was pressed");
+
+          keyInteraction.offKeyPress(aCode, aCallback1);
+          aCallback1Called = false;
+          aCallback2Called = false;
+          $target.simulate("keydown", { keyCode: aCode });
+          assert.isFalse(aCallback1Called, "callback 1 for \"a\" was disconnected from the interaction");
+          assert.isTrue(aCallback2Called, "callback 2 for \"a\" is still connected to the interaction");
```

Comment:
I don't see the purpose of this block here.  The name of this unit test is "multiple keyPress callbacks are possible" but this block here refers to the removal of keyPress callbacks.  Either the unit test needs to be split up or it needs to be renamed